src/main/java/net/imagej/launcher/SplashScreen.java

```diff
@@ -59,20 +64,32 @@ public static void show() {
 		if (Boolean.getBoolean("java.awt.headless")) return;
 		final JWindow window = new JWindow();
 		splashWindow = window; // Save a non-AWT reference to the window.
+
+		// Get splash icon from current class loader
 		final ClassLoader classLoader = //
 			Thread.currentThread().getContextClassLoader();
-		final URL logoURL = classLoader.getResource(LOGO_PATH);
-		final ImageIcon imageIcon;
+		URL logoURL = classLoader.getResource(LOGO_PATH);
+
 		if (logoURL == null) {
 			// Look for images/icon.png on disk, as a fallback.
 			// For backwards compatibility with old Fiji installations.
 			final String parent = System.getProperty("imagej.dir") != null ? //
 				System.getProperty("imagej.dir") : ".";
 			final File logoFile = new File(parent, "images/icon.png");
 			if (!logoFile.exists()) return;
-			imageIcon = new ImageIcon(logoFile.getPath());
+			try {
+				logoURL = logoFile.toURI().toURL();
+			} catch (MalformedURLException e) {
+				e.printStackTrace();
+				return;
+			}
 		}
-		else imageIcon = new ImageIcon(logoURL);
+
+		// Find all splash icons on the classpath
+		Map<String, URL> splashs = FileUtils.findResources(".*", "splash", null);
```

Comment:
I think... we better not depend on scijava-common in this project. I'll give some thought to the best way to avoid it here. I think we can recapitulate a targeted subset of what the `FileUtils.findResources` method does.